src/networks.ts

```diff
@@ -64,6 +64,101 @@ export const networks: Networks = {
     protocolMultisig: undefined,
   } as Network,
   // TEST NETWORKS
+  //   zoraTestnetSepolia: {  // TODO: Not supported yet
```

Comment:
@alexanderattar i think these can be added. I noticed you said layer zero does not support these yet but we can leave lzendpoint as null.